neurow/lib/sse_monitor.ex

```diff
@@ -2,15 +2,15 @@ defmodule SSEMonitor do
   require Logger
   use GenServer
 
-  def start_link(conn) do
-    GenServer.start_link(__MODULE__, conn)
+  def start_link(init) do
```

Comment:
The SSE endpoint continue to pass the current connection when it start the SSE monitor, maybe we can remove it.